flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -87,6 +89,42 @@ function setup_kubernetes_for_linux {
     sudo sysctl fs.protected_regular=0
 }
 
+function retry_times_with_download {
```

Comment:
shouldn't we be able to generalize it even more? We could move the actual download logic into this function as well. The only parameter that should be passed would be the URL of the artifact. WDYT?

Reply:
I was thinking about something like that:
```bash
function retry_times_with_download {
    if [[ "$#" != 1 ]]; then
       echo "Fatal error: No parameter or too many parameters passed: $@"
       exit 1;
    fi

    local download_url="$1"
    if ! retry_times ${RETRY_COUNT} ${RETRY_BACKOFF_TIME} "wget -nv ${download_url}"; then
      echo "ERROR: Download failed repeatedly after ${RETRY_COUNT} tries. Aborting..."
      exit 1
    fi
}
```

Reply:
Currently, CI is failing because of the changes.